Firebase/DynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.0.4
+- [fixed] Removed references to UIWebView to comply with App Store Submission warning. (#3723)
```

Comment:
Should be 3722 for the issue.

Thanks!

Reply:
Whoops, good call, will follow up.